pkg/util/http/links/links.go

```diff
@@ -0,0 +1,20 @@
+package links
+
+import "regexp"
+
+var linkRegex = regexp.MustCompile(`\<(.+?)\>\s*;\s*rel="(.+?)"(?:\s*,\s*)?`)
+
+func ParseLinks(header string) map[string]string {
```

Comment:
Doc for what this does.  It's no obvious to me why someone else would ever use this.   Why not just locate it with the only caller?